webapp/src/shared/components/Button/Button.css

```diff
@@ -12,6 +10,16 @@
   justify-content: center;
 }
 
+.button-large {
+  width: 15rem;
+  border-radius: 0.5rem;
+}
+
+.button-small {
+  width: 3rem;
+  border-radius: 50%;
```

Comment:
Not sure if yours, but button width in edit profile page seems to be broken:

![image](https://user-images.githubusercontent.com/57688116/200769590-8002cc64-5bb0-447a-97c7-af8d639e4a10.png)

Reply:
Yes, you broke the edit avatar page button back in september, and now in order to fix the broken edit avatar button, I broke every other button. Should be fixed now.